sdk/iot/Azure.Iot.Hub.Service/src/Customized/JobsRestClient.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service
+{
+    /// <summary>
```

Comment:
You don't want override reason in the xml doc comments. We don't want that showing up to the user or in online documentation.